apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,281 @@
+// Component for layout selection
+
+import React, { useState } from 'react'
+import {
+  Box,
+  Grid,
+  GridItem,
+  HStack,
+  Image,
+  Text,
+  VStack,
+  useToken,
+  Icon,
+} from '@chakra-ui/react'
+import { Button } from '@opengovsg/design-system-react'
+import { BiLeftArrowAlt, BiRightArrowAlt, BiShow } from 'react-icons/bi'
+import Preview from '~/features/editing-experience/components/Preview'
+import articleLayoutPreview from '~/features/editing-experience/data/articleLayoutPreview.json'
+import contentLayoutPreview from '~/features/editing-experience/data/contentLayoutPreview.json'
+
+type LayoutType = 'article' | 'content'
+
+export interface LayoutSelectionProps {
+  pageName: string
+  // preparing to make the trpc call to create page. required props
+  pageUrl: string
+  siteId: string
+  folderId: string
+}
+
+// export const createPageSchema = z.object({
+//   pageName: z.string(),
+//   pageTitle: z.string(),
+//   // TODO: add the actual layouts in here
+//   layout: z.enum(PAGE_LAYOUTS).default('content'),
+//   siteId: z.string(),
+//   // NOTE: implies that top level pages are allowed
+//   folderId: z.string().nullable(),
+// })
+
+const LAYOUT_DATA: {
+  layoutDisplayName: string
+  layoutTypename: LayoutType
+  layoutDescription: string
+  imageSource: string
+  previewJson: JSON
+}[] = [
+  {
+    layoutDisplayName: 'Default',
+    layoutTypename: 'content',
+    layoutDescription: 'This is the most basic layout for your content.',
+    imageSource: '/assets/layout-card/default_layout_card.png',
+    previewJson: contentLayoutPreview,
+  },
+  {
+    layoutDisplayName: 'Article',
+    layoutTypename: 'article',
+    layoutDescription:
+      'Designed for the perfect reading experience. Use this layout for text-heavy content, such as news, press releases, and speeches',
+    imageSource: '/assets/layout-card/article_layout_card.png',
+    previewJson: articleLayoutPreview,
+  },
+]
+
+// TODO: Make this headless by getting the LAYOUT_DATA from the schema. Find somewhere in the schema for layoutDescription & image(fetch this too or generate it)
+function LayoutSelection(props: LayoutSelectionProps): JSX.Element {
+  // need state here to keep track of selected layout
+  const [selectedLayout, setSelectedLayout] = useState(LAYOUT_DATA[0]!)
+  return (
+    <>
+      {/* This is the floating header */}
+      <VStack position="fixed" h="100%">
+        <VStack p="1.5rem 2rem" w="full" gap="0.5rem" alignItems="flex-start">
+          <Button variant="link">
+            <Icon as={BiLeftArrowAlt} />
+            Back to all pages
+          </Button>
+          <HStack p="0" justifyContent="space-between" w="100%">
+            <Text textStyle="h4" color="base.content.default">
+              {`Choose a layout for "`}
+              {props.pageName}
+              {`"`}
```

Comment:
nit: for readability

```suggestion
              {`Choose a layout for "${props.pageName}"`}
```